docs/specifications/tests/DNSSEC-TP/dnssec17.md

```diff
@@ -0,0 +1,213 @@
+# DNSSEC17: Validate trust from DS to CDS and CDNSKEY
+
+## Test case identifier
+**DNSSEC17**
+
+## Objective
+
+CDS and CDNSKEY record types are defined in [RFC 7344] and [RFC 8078].
+Both record types are optional in a zone. The objective of this test
+case is to verify that they are there is a correct chain of trust
+from DS, in the parent zone to the CDS and CDNSKEY RRsets
+([RFC 7344][RFC 7344, section 4.1], section 4.1).
+
+This Test case is only relevant if 
+* The *Child Zone* has either CDS or CDNSKEY record or both.
+* The parent zone has a DS RRset for the *Child Zone*.
+
+If a name server has issues covered by [Basic04] (basic name server
+issues) no messages will be outputted from this test case.
+
+It is assumed that the Child Zone has been tested by [DNSSEC15] and
+[DNSSEC16] and that the servers still give the same responses.
+
+## Inputs
+
+* "Child Zone" - The domain name to be tested.
+* "Test Type" - The test type with value "undelegated" or "normal".
+* "Undelegated DS" - The DS record or records submitted (only if
+  Test Type is undelegated).
+
+## Summary
+* If no CDS or CDNSKEY records are found, this test case is not run.
+* If no DS records are found at parent, this test case is not run.
+* [ERROR] message if the CDS RRset is not signed with the DNSKEY that
+  a DS record points at.
+* [ERROR] message if the CDNSKEY RRset is not signed with the DNSKEY
+  that a DS record points at.
```

Comment:
not "the DNSKEY" but "a DNSKEY". The DS set can point out more than one key.

Reply:
Yes, but the text is "a DS record points at", so it is per DS record.

Reply:
It is meant to be "the DNSKEY record". I can clarify.